CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. 
+
+## [2.0.0]
+
+### Features
+
+* Updated packages and namespace. Moved rudder_analytics to rudderstack.analytics
+* Added gzip compression
+* Added "channel" field to events
+* RuntimeError on message size exceeding 32kb
+* Rename flush_at to upload_size, flush_interval as upload_interval
```

Comment:
these need to be moved into a breaking changes section along with the rename of data_plane_url to dataPlaneUrl.

Double check that no othr options changes keyname.

We also need to mention for gzio we need server version 1.4 and above in docs